_posts/2020-07-10-removing-negative-words

```diff
@@ -0,0 +1,86 @@
+
+But we understand that is not enough. This is not an action that effects change in its own. It's part of a larger commitment. In this article, I'll explain the context and motivation for this change and how it fits with our core values and log term vision as a company.
+
+<!--more-->
+
+## Context
+
+As far as the software community is concerned, efforts to move away from such terms have been underway for quite some time now. In 2014, [Drupal](https://www.drupal.org/) opted for the use of "primary / replica" instead of "master / slave" and, in the same year, [Django] (https://www.djangoproject.com/) opted to use "leader / follower". Then in 2018 the decision was made to remove the terminology from Python, same year that the IETF (Internet Engineering Task Force) published their [Terminology, Power and Oppressive Language](https://tools.ietf.org/id/draft-knodel-terminology-00.html#rfc.section.1.1) memo arguing for the terminology change and offering alternatives to be use instead of the offensive terms.
+
+Triggered by current events, the discussion has resurfaced, and recently several companies, big and small, and communities have been making the change (examples include OpenZFS and Go) or making a commitment to change, with GitHub's CEO claiming [in a Tweet](https://twitter.com/natfriedman/status/1271253144442253312) that his team has already been working on transitioning from "master" to "main" for default branch structure. The Android and Chrome teams have also been determined to replace "whitelist / blacklist" with "allowlist / blocklist".
+
+Not everyone in the community agrees with the change though. When Python moved away from the terminology in 2018, there was a heated discussion over it, with several members of the community questioning the change and it's real impact. Before that, the issue had already divided the Redis community.
+
+It is necessary to keep in mind that in order to completely replace the terms in question, there needs to be an industry wide decision to stop using them. As long as there are still projects who haven't moved away from the terminology, it won't be completely wiped out. A good example is Kubernetes. Although it uses "replicas", "workers", and "minions" instead of "slaves", [as pointed out by Sinclair Im in a Washington Post editorial](https://www.washingtonpost.com/opinions/2020/06/12/tech-industry-has-an-ugly-master-slave-problem/), "its code repository still contains more than 200 lines that use ‘slave.’ Because its code must talk to others, even Kubernetes can’t completely avoid the terminology".
+
+As such, it is important to have the discussion around the necessity of this change, it's motivation and it's impact. Of course large corporations can simply get their executives to issue a top-bottom order forbidding the use of these terms. But it might prove difficult to convince independent developers and smaller projects to adopt the new terms, and quickly.
```

Comment:
```suggestion
As such, it is important to have the discussion around the necessity of this change, its motivation and its impact. Of course large corporations can simply get their executives to issue a top-bottom order forbidding the use of these terms. But it might prove difficult to convince independent developers and smaller projects to adopt the new terms, and quickly.
```